src/components/PeopleList.tsx

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonRow } from './PersonRow';
+
+export const PeopleList = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loadingError, setLoadingError] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  const getPreparedPeople = (loadedPeople: Person[]) => {
+    return loadedPeople.map(person => {
+      return {
+        ...person,
+        mother: people
+          .find(person1 => person1.name === person.motherName),
+        father: people
+          .find(person1 => person1.name === person.fatherName),
+      };
+    });
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const loadedPeople = await getPeople();
+
+        setPeople(loadedPeople);
+      } catch (error) {
+        setLoadingError(true);
```

Comment:
When catching an `error`, it is better to have an understanding of exactly what went wrong and output it to the user or to the console: 
```
setLoadingError(`Something went wrong: ${error.message}`)
```